api/envoy/api/v2/core/base.proto

```diff
@@ -150,6 +150,17 @@ message RuntimeUInt32 {
   string runtime_key = 3 [(validate.rules).string = {min_bytes: 1}];
 }
 
+// Runtime derived bool with a default when not specified.
```

Comment:
Can you remove this from this change or add explicit tests for utility functions for this proto?

Reply:
Just removing it.